monitors/onlinemon/src/EUDAQMonitorCollection.cc

```diff
@@ -32,7 +32,7 @@ void EUDAQMonitorCollection::Write(TFile *file) {
     std::cerr << "EUDAQMonitorCollection::Write File pointer is NULL" << endl;
     exit(-1);
   }
-  if (gDirectory != NULL) // check if this pointer exists
+  if (gDirectory) // check if this pointer exists
```

Comment:
`!= nullptr`?

Reply:
Nope, not possible since root 6.24.

Reply:
because it's not a pointer?

Reply:
Looking at the redefinition, a

```cpp
gDirectory() != nullptr
```

should work

Reply:
They have wrapped this into some construction to be threadsave, which overrides the operator != twice. 
TDirectory* dir = nullptr
If(gDirectory!=dir)

Would work, but they have a direct check implemented now 🤷

Reply:
Whatever... :)

Reply:
It works....